api/models/group.py

```diff
@@ -37,10 +51,31 @@ class Group(models.Model):
     saturday = models.TimeField(blank=True, null=True)
```

Comment:
Because time slots are different from exact times. A group is in time slot 8-11, but once it's confirmed, the teacher arranges a specific time with their students, and they agree that the class begins e.g. at 10:30. And the coordinator should know the exact time, of course.

Reply:
Marking as resolved, as we discussed this yesterday.